assemblies/assembly_test-scenarios/main.adoc

```diff
@@ -31,8 +31,8 @@ include::{drools-dir}/AuthoringAssets/test-designer-con.adoc[leveloffset=+1]
 include::{drools-dir}/AuthoringAssets/test-designer-data-objects-import-proc.adoc[leveloffset=+2]
 include::{drools-dir}/AuthoringAssets/test-designer-test-scenario-import-proc.adoc[leveloffset=+2]
 include::{drools-dir}/AuthoringAssets/test-designer-save-test-proc.adoc[leveloffset=+2]
-include::{drools-dir}/AuthoringAssets/test-designer-delete-test-proc.adoc[leveloffset=+2]
-include::{drools-dir}/AuthoringAssets/test-designer-rename-test-proc.adoc[leveloffset=+2]
+//include::{drools-dir}/AuthoringAssets/test-designer-delete-test-proc.adoc[leveloffset=+2]
+//include::{drools-dir}/AuthoringAssets/test-designer-rename-test-proc.adoc[leveloffset=+2]
```

Comment:
Shouldn't we remove these lines (rather than comment) and probably the referenced files?